iphone/TitaniumKit/TitaniumKit/Sources/API/TiBlob.h

```diff
@@ -75,6 +75,10 @@ JSExportAs(imageWithRoundedCorner,
            : (NSNumber *)optionalBorderSize);
 - (TiBlob *)imageWithTransparentBorder:(NSUInteger)size;
 - (NSString *)toString; // FIXME This doesn't seem to override the JS impl. I think we need to find a way to modify the property post-init to override it!
+JSExportAs(compare,
```

Comment:
The image comparison stuff is now baked into our test suite, so we do not need a native implementation on Ti.Blob here.